cohort-engine/src/main/java/com/ibm/cohort/engine/measure/MeasureVersion.java

```diff
@@ -0,0 +1,82 @@
+/*
+ * (C) Copyright IBM Corp. 2021, 2021
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+package com.ibm.cohort.engine.measure;
+
+import java.util.Optional;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+
+public class MeasureVersion implements Comparable<MeasureVersion> {
```

Comment:
This seems like logic that might be useful in other places. I would consider naming it SemanticVersion and maybe putting it either the parent package or maybe a util package.

Reply:
Thanks for the suggestion. I went the subproject route and will bring it up with the team at our next meeting.